src/core/render/cursor.ts

```diff
@@ -546,7 +546,11 @@ const getSlots = (componentCtx: ComponentCtx | undefined, hostElm: Element): Slo
 };
 
 const handleStyle: PropHandler = (ctx, elm, _, newValue) => {
-  setAttribute(ctx, elm, 'style', stringifyClassOrStyle(newValue, false));
+  const style = stringifyClassOrStyle(newValue, false);
+  if (Object.keys(style).length > 0) {
```

Comment:
I think it should be:
```
  if (style.length > 0) {
```
since. its already a string.

Actually, why do we need this if statement at all, I can see a problem that styles never get removed.

Reply:
@manucorporat, yes you are right, for some reason saw that as object. My idea was to handle the case where style would be an empty string: 

<div style={{display: undefined}}></div> to <div style=''></div>.